docs/concepts/conda_pypi.md

```diff
@@ -51,13 +51,13 @@ Because Pixi supports both ecosystems, it currently needs two different solvers
     Because resolvo is written to support both ecosystems, it is possible to use it for PyPI packages as well, but this is not yet implemented.
 
 Because of these two solvers, we need to make a decision which solver to run first.
-This is the conda-first approach, which means that we first solve the conda dependencies and then the PyPI dependencies.
+Our decision is the conda-first approach, which means that we first solve the conda dependencies and then the PyPI dependencies.
```

Comment:
```suggestion
Because the PyPI package needs a base environment to install into, we need to use the conda-first approach, which means that we first solve the conda dependencies and then the PyPI dependencies.
```

Or something like that, it's not really an option to do it the other way around. But please help me get that clear!